### ISSUE TITLE
Default Font

### ISSUE THREAD

**Issue by 73** [liferooter/textpieces#35]
Hi, 

thank you for your work. It makes my life better. 

Textpieces does not pick my default monospace font from gsettings. `/org/gnome/desktop/interface/monospace-font-name`
Is that intentional?

I am running version 2.3.3 from Flathub. 

Thanks.

**Comment by liferooter** [comment]
Thank you for your issue!

**Comment by liferooter** [comment]
No, it wasn't intentional

**Comment by liferooter** [comment]
## Some technical notes

Manually load monospace font name from `/org/gnome/desktop/interface/monospace-font-name` is little hacky. Also it may not work on non-gnomish systems. I was surprised that GTK has no good way to load default monospace font from this schema. Instead it, GTK has fontconfig's `Monospace` default font. So I decided to do the same thing that GNOME Builder does. I'll add ability to select font, but default font will be `Monospace`.

## TL;DR

I'll add font option to settings, but default font will be system's default `Monospace`, not default monospace from `/org/gnome/desktop/interface/monospace-font-name`.

**Comment by 73** [comment]
I see. That is perfect, too.

**Comment by liferooter** [comment]
This feature will appear in Flathub's Text Pieces in one or two days I guess when I'll publish new version
